### ISSUE TITLE
always doesn't work 

### ISSUE THREAD

**Issue by idmytro** [ChrisShank/xstate-vue2#8]
This simple machine doesn't work with xstate-vue2, but works with @xstate/vue

```ts
const simpleMachine = createMachine({
  id: 'simple',
  initial: 'idle',
  states: {
    idle: {
      always: [
        {actions: 'log', target: 'active'}
      ],
    },
    active: {}
  }
},
{
  actions: {
    log: () => console.log('log')
  }
})
```

**Comment by ChrisShank** [comment]
Peculiar, I think I can reproduce this https://stackblitz.com/edit/vite-9wsjog?file=main.js

> Cannot set properties of undefined (setting 'value')

Seems like `listener` is called before the `state` variable is initialized. Should be easy to fix :)

**Comment by ChrisShank** [comment]
Okay so this was a regression caused by the last [bug fix](https://github.com/ChrisShank/xstate-vue2/issues/6). `@xstate/vue` doesn't have this bug because that other bug hasn't been patched. I'll try to fix this regression tomorrow, thanks for reporting!

**Comment by daniel-n-dream** [comment]
I also get the error `Cannot set properties of undefined (setting 'value')` with a Machine that uses Always transitions a lot, is there any possible workaround or update on this @ChrisShank ?

**Comment by ChrisShank** [comment]
Fixed in v0.3.2

See the reproducer with the latest version https://stackblitz.com/edit/vite-vqm8hy?file=package.json
